x-pack/plugins/enterprise_search/server/lib/fetch_search_results.ts

```diff
@@ -21,7 +21,7 @@ export const fetchSearchResults = async (
     from,
     index: indexName,
     size,
-    ...(!!query ? { q: JSON.stringify(query) } : {}),
+    ...(!!query ? { q: query.replace(/"/g, '\\"') } : {}),
```

Comment:
I'm good with this, but should we do any follow-up to ensure the query string is sanitized further?

Reply:
@TattdCodeMonkey That's a great question. Did you have anything specific in mind? 

@sphilipse seemed to think that was sufficient because you can't really inject anything into the query at that point plus it's already behind authorization. I really dislike the idea of ad-hoc manual safety String mutations for several reasons. We could consider adding a new issue to re-do the search form so that Stringify doesn't force everything to use multi-match, but I'm not really sure what the priority is. WDYT?

Reply:
@kderusso The call is a `query_string` query, right? Could you describe how the multi-word query fails due to the index not supporting multi-match, e.g. show the error message? Maybe we can fix it in a different way.

My concern is putting the expression in quotes forces a phrase match, which differs from the actual full text matching experience (OR semantics by default). But I agree this is low priority.

Reply:
@demjened Can you please help me better understand your concern here? `JSON.stringify` was quoting the query and forcing a multi-match query to be run. This removes this default behavior.

Reply:
Ah, I thought `query` was already coming in quoted from the UI. But then why the need for `.replace()`? Can't we just pass `q: query` as-is?

Reply:
Users can still input a quote in the front-end, which will force a multi-match if we don't escape them.

Quick question: do we also need to escape single quotes, or only double quotes?

Reply:
@sphilipse I have verified that single quotes don't need to be escaped, using them doesn't error. 

@demjened If we use double quotes it will force the multi-match phrase query and error. Escaping them solves this.